README.md

```diff
@@ -102,6 +102,14 @@ tabline works and looks like.
 The following is the list of available options, that should be set in your
 Vimrc file, using `let <optiona-name> = <option-value>`:
 
+* `g:buffet_always_show_tabline` - if set to `0`, the tabline will only be shown if
+  there is more than one buffer or tab open.
+
+  Default:
+  ```
+  let g:buffet_always_show_tabline = 1
+  ```
+
```

Comment:
There is always at least one tab open at any time, so you can limit this to just "more than one buffer open".

Reply:
I mean, it can never be 0 tabpages.